skxray/core.py

```diff
@@ -603,27 +603,22 @@ def bin_1D(x, y, nx=None, min_x=None, max_x=None):
     return bins, val, count
 
 
-def pixel_to_radius(shape, calibrated_center, pixel_size=None):
+def radial_grid(center, shape, pixel_size=None):
     """
-    Converts pixel positions to radius from the calibrated center
+    Make a grid of radial positions.
 
     Parameters
     ----------
-    shape : tuple
-        The shape of the image (nrow, ncol) to warp
-        the coordinates of
+    center : rr, cc : tuple
```

Comment:
Can you make sure this wraps into vistrails correctly?  I do not think that this will pass through numpydoc correctly and almost certainly will fail on my parsing tools.